.github/workflows/release_policy.yml

```diff
@@ -0,0 +1,104 @@
+name: Release Policy
+
+on:
+  workflow_dispatch:
+    inputs:
+      owner:
+        description: "Owner"
+        required: true
+        type: string
+      repo:
+        description: "Repository"
+        required: true
+        type: string
+      tag:
+        description: "Tag to release"
+        required: true
+        type: string
+  repository_dispatch:
+    types: [release-policy]
+
+jobs:
+  release-policy:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - name: Checkout current repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+
+      - name: Extract repository info
+        id: repo-info
+        run: |
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            echo "owner=${{ github.event.inputs.owner }}" >> $GITHUB_OUTPUT
+            echo "repo=${{ github.event.inputs.repo }}" >> $GITHUB_OUTPUT
+            echo "tag=${{ github.event.inputs.tag }}" >> $GITHUB_OUTPUT
+          else
+            echo "owner=${{ github.event.client_payload.owner }}" >> $GITHUB_OUTPUT
```

Comment:
I would use `on.workflow_call.inputs` and then just use `${{ inputs.owner }}` and so without ifs, which seem equivalent for me. (Example: https://github.com/kubewarden/kubewarden-end-to-end-tests/blob/main/.github/workflows/e2e-tests.yml#L60)

Reply:
`workflow_call` is used in the context of reusable actions, allowing this action to be executed within the policy repository. This is not the case here, since the goal is to enable triggering from another repository while running the release action in this one. We use this approachin the Helm chart release flow:  

- [trigger from kwctl](https://github.com/kubewarden/kwctl/blob/df1d4134949afe3d22ee8fd994340ae36a2f897e/.github/workflows/release.yml#L150) 
- [Helm charts update workflow](https://github.com/kubewarden/helm-charts/blob/8ac84b0fcfade9bfd8cc326b66d98b0156380160/.github/workflows/update-charts.yml#L34)

Additionally, I've included `workflow_dispatch`, which allows for manual triggering of the action when needed.